mne/preprocessing/tests/test_asr.py

```diff
@@ -0,0 +1,38 @@
+# Authors:  Dirk Gütlin <dirk.guetlin@gmail.com>
```

Comment:
don't put a big file like mne/preprocessing/tests/data/matlab_asr_data.mat in the repo. It should be in the mne-testing repo

Reply:
Yes, as mentioned in the initial commit, this is just a preliminary thing. In case we want to keep this file in the testing procedure, I'll move it to the testing data repo...